src/main/java/com/redhat/devtools/intellij/lsp4ij/operations/completion/LSContentAssistProcessor.java

```diff
@@ -76,6 +77,10 @@ public void fillCompletionVariants(@NotNull CompletionParameters parameters, @No
                 }
 
             }
+        } catch (ProcessCanceledException ignored){
+            LOGGER.debug("Completion was canceled");
+            //nothing left to do here, return early
+            return;
```

Comment:
I wonder if we should throw this exception?

Reply:
yeah that works